federation/cluster/common.sh

```
@@ -87,7 +87,18 @@ function create-federation-api-objects {
       fi
     fi
 
-    export FEDERATION_SERVICE_CIDR=${FEDERATION_SERVICE_CIDR:-"10.10.0.0/24"}
+    if [[ "${KUBERNETES_PROVIDER:-}" == "gce" || "${KUBERNETES_PROVIDER:-}" == "gke" ]]; then
+      PD_PLUGIN_NAME='kubernetes.io/gce-pid'
+    elif [[ "${KUBERNETES_PROVIDER:-}" == "aws" ]]; then
+      PD_PLUGIN_NAME='kubernetes.io/aws-ebs'
+    else
+      echo "not sure how to provision PD volume for ${KUBERNETES_PROVIDER:-}"
```

I would error out here, rather than just print a warning.